include/geometrycentral/surface/polygon_soup_mesh.h

```diff
@@ -3,18 +3,21 @@
 #include <cstdlib>
 #include <fstream>
 #include <memory>
+#include <regex>
```

Comment:
looks like this isn't used, right? can we remove it?

Reply:
Yeah, this was left over from a previous version of the code.